exasol/nb_connector/slct_manager.py

```diff
@@ -40,20 +43,29 @@
 SLC_RELEASE_TAG = "9.6.0"
 
 
+# Maybe we could find a better name for this class?
+#
+# E.g.
+# - SlcCoordinates
+# - SlcMetadata
+# - SlcLocation
```

Comment:
The purpose of the class is to centralize the path names of the SLC.
I would be fine with `SlcLocation`, or maybe `SlcPaths`....

Reply:
I like `SlcPaths`! 
3 - 2 - 1 - sold!